packages/fluentui/react-bindings/src/styles/resolveStyles.ts

```diff
@@ -210,8 +210,8 @@ export const resolveStyles = (
           }
 
           if (process.env.NODE_ENV !== 'production' && isDebugEnabled) {
-            resolvedStylesDebug[slotName] = resolvedStyles[slotName]['_debug'];
-            delete resolvedStyles[slotName]['_debug'];
+            resolvedStylesDebug[slotName] = (resolvedStyles[slotName] as any)['_debug'];
```

Comment:
it's kinda looking like we need an internal type that has `_debug` as a propertry